Introduce cross-cluster replication API docs (#34726)
elastic/elasticsearch: docs/reference/ccr/apis/auto-follow/delete-auto-follow-pattern.asciidoc

[role="xpack"]
[testenv="platinum"]
[[ccr-delete-auto-follow-pattern]]
=== Delete Auto-Follow Pattern API
++++
<titleabbrev>Delete Auto-Follow Pattern</titleabbrev>
++++

Delete auto-follow patterns.

==== Description

This API deletes configured auto-follow patterns.

==== Request

//////////////////////////

[source,js]
--------------------------------------------------
PUT /_ccr/auto_follow/my_auto_follow_pattern
{
  "leader_cluster" : "leader_cluster",
  "leader_index_patterns" :
  [
    "leader_index"
  ],
  "follow_index_pattern" : "{{leader_index}}-follower"
}
--------------------------------------------------
// CONSOLE
// TEST[setup:leader_cluster]
// TESTSETUP

//////////////////////////

[source,js]
--------------------------------------------------
DELETE /_ccr/auto_follow/<auto_follow_pattern_name>
--------------------------------------------------
// CONSOLE
// TEST[s/<auto_follow_pattern_name>/my_auto_follow_pattern/]

Here, the auto-follow patterns associated with the
<<modules-remote-clusters,remote cluster>> specified by `<leader_cluster>`
will be deleted.

> 

Reviewer: ```suggestion

==== Path Parameters

`auto_follow_pattern_name` (required)::
  (string) Specifies auto-follow patterns that you want to delete.

```

==== Example

> The following is an example of using the delete auto-follow pattern API.

Reviewer: ```suggestion
The following example deletes an auto-follow pattern named `my_auto_follow_pattern`:
```
